ldfparser/parser.py

```diff
@@ -10,6 +10,7 @@
 
 class LDF:
 	def __init__(self):
+		self._source = None
```

Comment:
This may impact memory usage, it was only added so that the CLI can use both the LDF for queries as well as the dictionary for JSON export. Instead the parsing methods should allow to convert file to dictionary and dictionary to LDF object.